include/zephyr/drivers/espi_emul.h

```diff
@@ -116,20 +53,65 @@ struct espi_emul {
 	/** Parent emulator */
 	const struct emul *parent;
 	/** API provided for this device */
-	const struct emul_espi_device_api *api;
+	const struct espi_emul_device_api *api;
 	/** eSPI chip-select of the emulated device */
 	uint16_t chipsel;
 };
 
-/** Definition of the eSPI controller emulator API */
-struct emul_espi_driver_api {
-	/* The struct espi_driver_api has to be first in
-	 * struct emul_espi_driver_api to make pointer casting working
+/** Data about the virtual wire */
+struct espi_emul_vw_data {
+	/* Virtual wire signal */
+	enum espi_vwire_signal sig;
+	/* The level(state) of the virtual wire */
+	uint8_t level;
+	/* The direction of the virtual wire. Possible values are
+	 * ESPI_MASTER_TO_SLAVE or ESPI_SLAVE_TO_MASTER
```

Comment:
Inclusive language, please rephrase to `controller`/`target`

Reply:
It's using the defines from `espi.h`, do we want to change them into enums?

Reply:
I would approve of converting the constants in `espi.h` to enums, but that's probably out of scope here.

Reply:
After some refactoring, they were redundant, so I've just removed this field and usage of these defines.